terasoluna-gfw-parent/pom.xml

```diff
@@ -577,6 +577,9 @@
     <cargo.container.id>tomcat9x</cargo.container.id>
     <cargo.container.url>http://archive.apache.org/dist/tomcat/tomcat-9/v${cargo.tomcat9.version}/bin/apache-tomcat-${cargo.tomcat9.version}.zip</cargo.container.url>
     <cargo.tomcat9.version>9.0.55</cargo.tomcat9.version>
+    <!-- == Lombok == -->
+    <lombok.version>1.18.24</lombok.version>
```

Comment:
If you make a note that it is aligned with Spring Boot Dependencies, you will not be confused later.

Reply:
dbbb76fe98177e82c00f390d4b407b6a33c643f8
added a comment.